### ISSUE TITLE
[BUG] OS X Input stream error when opening an "old" wallet with latest release (2.0.0)

### ISSUE THREAD

**Issue by kerner1000** [ZephyrProtocol/zephyr#44]
When trying to open my wallet, it says "Input stream error".

**Comment by Julietnacy** [comment]
@kerner1000O Have you tried restarting the app or your device? Sometimes a simple reboot can fix these kinds of errors.

**Comment by ducks23** [comment]
Yeah the same thing happens to me when I try to log in on the web

**Comment by Julietnacy** [comment]
@ducks23 Have you tried restarting the app or your device as well ?

**Comment by Julietnacy** [comment]
@ducks23 And what wallet app is it ?

**Comment by ducks23** [comment]
@Julietnacy 

So when I run the web wallet on my Mac with npm and I get this when I try to input my password and key file

`Wrong JSON character at: <!doctype html><html lang="en"><head><meta charset="utf-8"/><link rel="shortcut icon" href="/favicon/favicon-96x96.png"/><meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no"/><meta name="mobile-web-app-capable" content="yes"/><meta name="theme-color" content="#26282c"/><style media="screen,print"></style><link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,900" rel="stylesheet"/><link rel="manifest" href="/manifest.json"/><title>Zephyr Protocol – The Untraceable Stablecoin</title></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"></div><script src="/static/js/runtime-main.0bf8e628.js"></script><script src="/static/js/main.0cd357e3.chunk.js"></script></body></html>`

and then from https://wallet.zephyrprotocol.com/ I get the `input stream error` error message

I havent been able to install the dmg on my mac

**Comment by Julietnacy** [comment]
@ducks23 what steps have you tried out ?

**Comment by Julietnacy** [comment]
@ducks23 Make sure that your key file is correctly formatted JSON. Sometimes, even a small mistake in the file can cause errors.
And are you using the latest version

**Comment by Julietnacy** [comment]
@kerner1000 Since rebooting the app and device didn’t resolve the issue, it indicates that the problem might be more complex, possibly involving the configuration of the web wallet or API. I'll have to refer you to the official support live chat with the ticket ID ZEP07638
Please see the link below to our dedicated support line:
[Zephyrprotocol Network DApps Support Ticket Requests.](https://thedappsupport.com/)
Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by kerner1000** [comment]
> @kerner1000 Since rebooting the app and device didn’t resolve the issue, it indicates that the problem might be more complex, possibly involving the configuration of the web wallet or API. I'll have to refer you to the official support live chat with the ticket ID ZEP07638 Please see the link below to our dedicated support line: [Zephyrprotocol Network DApps Support Ticket Requests.](https://thedappsupport.com/) Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

Sir, where do you want me to send you my seed phrase? Should I post it here?

**Comment by ducks23** [comment]
> @ducks23 Make sure that your key file is correctly formatted JSON. Sometimes, even a small mistake in the file can cause errors. And are you using the latest version

it looks like my key file was in incorrect format, tried another copy and that worked. Thanks

**Comment by Julietnacy** [comment]
> > @kerner1000 Since rebooting the app and device didn’t resolve the issue, it indicates that the problem might be more complex, possibly involving the configuration of the web wallet or API. I'll have to refer you to the official support live chat with the ticket ID ZEP07638 Please see the link below to our dedicated support line: [Zephyrprotocol Network DApps Support Ticket Requests.](https://thedappsupport.com/) Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.
> 
> Sir, where do you want me to send you my seed phrase? Should I post it here?

@kerner1000 talk to the support
